src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/ApplyVQSR.java

```diff
@@ -81,7 +81,7 @@
  *   -R Homo_sapiens_assembly38.fasta \
  *   -V input.vcf.gz \
  *   -O output.vcf.gz \
- *   --ts_filter_level 99.0 \
+ *   --truth-sensitivity-filter-level 99.0 \
```

Comment:
@chandrans Might as well change the other one of these below on line 97.

Reply:
Thanks Chris! Done